torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19497,6 +19504,15 @@ def __init__(
         "_refs.nn.functional.elu",
         torch_opinfo_name="nn.functional.elu",
     ),
+    PythonRefInfo(
```

Comment:
We should make this (and the gelu OpInfo) elementwise unary. The easiest way to do this is probably to model the original OpInfo with a variant for the tanh approximation (that would require splitting this OpInfo, too)

Reply:
Can we do this in a later PR? This looks a little tricky.